dev/com.ibm.ws.security.fat.common.mp.jwt/publish/shared/config/builder_fatTestPorts.xml

```diff
@@ -1,4 +1,16 @@
-<server>
+<!--
+    Copyright (c) 2024 IBM Corporation and others.
```

Comment:
The copyright date for this should start when the file was added.

Reply:
Hard to find that - I try to put the creation date in, but it started in a different repo, and some of the history has been lost.